File: array/java/Array.java

public class Array {
    public static void main(String[] args) {
        int[] odds = { 1, 3, 5, 7, 9 };

        for (int i = 0; i < odds.length; i++) {
            System.out.println(
                String.format("  odds[%d] = %d", i, odds[i])
            );
        }
    }
}

// Array.java

File: array/c++/array.cpp

#include <iostream>

using namespace std;

int main(int argc, char** argv) {
    int odds[] = { 1, 3, 5, 7, 9 };

    for (int i = 0; i < sizeof(odds) / sizeof(int); i++) {
        cout << "  odds[" << i << "] = " << odds[i] << endl;
    }
}

// array.cpp
